discretize/View.py

```diff
@@ -581,14 +593,15 @@ def _plotCylTensorMesh(self, plotType, *args, **kwargs):
         # Just create a TM and use its view.
         from discretize import TensorMesh
 
-        vType = kwargs.pop('vType', None)
+        vType = kwargs.get('vType', None)
```

Comment:
Is there a reason this got changed to `get` from `pop`?

Reply:
yeah, for 'CC', we don't reassign vType, and it still needs to be passed to the tensor plot